RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -1735,6 +1735,7 @@ string safeString(item input);
 string safeString(monster input);
 void handleTracker(string used, string tracker);
 void handleTracker(string used, string detail, string tracker);
+void handleTracker(string used, string loc, string detail, string tracker);
```

Comment:
if this is meant to always be location any harm in having `location loc`? Optional.